instrumentation/instrument.h

```diff
@@ -48,8 +49,10 @@ void bx_instr_before_execution(unsigned cpu, bxInstruction_c *i);
 #define BX_INSTR_EXIT(cpu_id) \
   bx_instr_exit(cpu_id)
 
-#define BX_INSTR_LIN_ACCESS(cpu_id, lin, phy, len, rw) \
-  bx_instr_lin_access(cpu_id, lin, phy, len, rw)
+#define BX_INSTR_LIN_ACCESS(cpu_id, lin, phy, len, memtype, rw) \
+                bx_instr_lin_access(cpu_id, lin, phy, len, memtype, rw)
+
+#define BX_INSTR_PHY_ACCESS(cpu_id, phy, len, memtype, rw)
```

Comment:
Could you move this #define to its corresponding location in the "Stubs for the rest of the macros." section? This section is only for hooking up the instrumentation functions (as the "Implemented instrumentation." name suggests).